migrations/versions/b25352d6508f_field_guides.py

```diff
@@ -0,0 +1,44 @@
+"""field guides
+
+Revision ID: b25352d6508f
+Revises: 3a11c87ddfd7
+Create Date: 2018-01-29 10:21:24.851812
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'b25352d6508f'
+down_revision = '3a11c87ddfd7'
+branch_labels = None
+depends_on = None
+
+import json
+import os
+
+import sqlalchemy as sa
+from alembic import op
+
+import tsammalex
+
+VPREFIX = 'https://cdstar.shh.mpg.de/bitstreams/'
+
+
+def upgrade():
+    with open(os.path.join(os.path.dirname(tsammalex.__file__),
+                           'static', 'field_guides.json')) as fp:
```

Comment:
I would make the migration stand-alone by pasting the JSON as python `dict` into the migration module.